paddle/go/optimizer/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
That's a good point. In the final form we should share optimizer when running locally and on pserver. Let's move to `/repo/paddle/optimizer`.